extensions/cornerstone/src/Viewport/Overlays/CustomizableViewportOverlay.tsx

```diff
@@ -132,6 +134,20 @@ function CustomizableViewportOverlay({
     };
   }, [viewportId, viewportData, voi, element]);
 
+  const annotationModified = useCallback(evt => {
```

Comment:
This shouldn't be done this way; please remove it. I guess displaying it inside the Panel is easiest for now.